NeuralAmpModeler/NeuralAmpModeler.cpp

```diff
@@ -66,40 +66,41 @@ class IVUpdateableLabelControl : public IVLabelControl
 // Styles
 const IVColorSpec activeColorSpec{
   DEFAULT_BGCOLOR, // Background
-  PluginColors::NAM_1, // Foreground
-  PluginColors::NAM_2.WithOpacity(0.4f), // Pressed
-  PluginColors::NAM_3, // Frame
```

Comment:
Are these old colour constants still used anywhere else? If not, the declarations could be removed.

Reply:
NAM_1 is actually used, yes...
the other ones not i think. does it hurt to leave them ?

Reply:
I don't mind leaving them. Nit-level